backend/server.py

```diff
@@ -168,15 +180,73 @@ def delete_source(translation_source_id: int, _: dict = Depends(get_user_info)):
     translations = list(Sources.select().where(Sources.original_source_id == source.original_source_id))
     source_ids_to_delete = [translation_source_id]
     if len(translations) == 1:
```

Comment:
Is this condition still correct?
After migration we can have multiple origin sources for translation.